package.json

```diff
@@ -58,13 +58,15 @@
     "@angular/compiler": "^5.2.6",
     "@angular/compiler-cli": "^5.2.6",
     "@angular/core": "^5.2.6",
+    "@angular/flex-layout": "2.0.0-beta.10-4905443",
```

Comment:
There is new version

Reply:
There is, but we've checked experimentally and this one was the newest working.